overviewwidget.cpp

```diff
@@ -88,3 +93,20 @@ void OverviewWidget::modelUpdated()
 {
     m_treeView->expandAll();
 }
+
+void OverviewWidget::onItemActivated(const QModelIndex &index)
+{
+    if (!index.isValid())
+        return;
+
+    ModelTreeNode *node = static_cast<ModelTreeNode *>(index.internalPointer());
+
+    // it is assumed, that if item has children, user wants to expand it, even though sometimes it could be jumped to.
```

Comment:
This is right - I will try to fix the situation. Simple removal might not be enough, because than, double clicking on expandable item would both expand it, and move cursor to different position in source code, which might be unexpected from the user point of view.